src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -95,7 +98,12 @@ var editCommentHandler = function(e) {
         success: function(data) {
             if (!data.isError) {
                 if(isInCommentsPage()) {
-                    panelHeading.click();
+                    var user = formObject.find("[name='user']").val();
+                    var courseId = formObject.find("[name='courseid']").val();
+                    var fsName = formObject.find("[name='fsname']").val();
+                    var fsIndx = formObject.find("[name='fsindex']").val();
+                    
+                    loadFeedbackResponseComments(user, courseId, fsName, fsIndx, panelHeading, false);
```

Comment:
This change seems to be repeated twice. Any chance of abstracting it out to reduce duplication?